lib/matplotlib/collections.py

```diff
@@ -1462,12 +1462,14 @@ def __init__(self,
 
         .. plot:: gallery/lines_bars_and_markers/eventcollection_demo.py
         """
-
+        if positions is None:
+            raise ValueError('positions must be an array-like object')
+        positions = np.array(positions, copy=True)
```

Comment:
Technically np.array already copies by default (and there's likely a ton of places in the codebase which rely on that), so just a comment `# force a copy` is enough.

Reply:
I think having `copy=True` makes it much more clear what is going on, but happy to add a comment too